### DISCUSSION TITLE
VLC don't show up metadata (title, image, ....)

### DISCUSSION THREAD

**Discussion by itzboterberg** [AzuraCast/AzuraCast#7612] [General]
Hi I am trying on vlc mobile and on pc to get metadata I have already used with `http` but still no resultatat

https://servwave.org/rstore/1w6embjus6jhy2dndctvd3oi0v1hqjhhrooot4ld.png

**Comment by bombelman** [comment]
the image won't show in VLC.
the artist/title metadata will show in the titlebar (cropped from your screenshot) or in the playlist view.

probeer het eens ;-)
